quetz/main.py

```diff
@@ -749,7 +749,10 @@ def patch_channel(
             )
 
     for attr_, value_ in user_attrs.items():
-        setattr(channel, attr_, value_)
+        if attr_ == "metadata":
```

Comment:
wouldn't it be better to merge with the exisitng metadata instead of replacing it. using something like this:

```
old_meta = json.loads(channel.metadata)
old_metadata.update(new_metadata)
```

Reply:
yes perhaps, @wolfv what do you think?

Reply:
agreeing with btel!

Reply:
Done